src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy", sans-serif;
+  font-weight: bold;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+}
+
+.form-big {
+  position: relative;
+  margin-bottom: 30px;
+}
+
+.img-big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 26px;
+  left: 24px;
+}
+
+.input-big {
+  width: 100%;
+  height: 70px;
+  padding: 0 62px;
+  box-sizing: border-box;
```

Comment:
Don't repeat yourself. Ypu wrote
`* {
  box-sizing: border-box;
}`

above in your code and repeating it in many selectors